docs/lib/getting-started/fragments/android/integrate.md

```diff
@@ -6,69 +6,43 @@ try {
     Amplify.configure(getApplicationContext());
     Log.i("AmplifyGetStarted", "Amplify is all setup and ready to go!");
 } catch (AmplifyException exception) {
-    Log.e("AmplifyGetStarted", exception.getMessage());
+    Log.e("AmplifyGetStarted", "Failed to initialize Amplify", exception);
 }
 ```
 
 b. First add some data to your backend:
 
 ```java
 Task task = Task.builder().title("My first task").description("Get started with Amplify").build();
-
-Amplify.API.mutate(task, MutationType.CREATE, new ResultListener<GraphQLResponse<Task>>() {
-    @Override
-    public void onResult(GraphQLResponse<Task> taskGraphQLResponse) {
-        Log.i("AmplifyGetStarted", "Added task with id: " + taskGraphQLResponse.getData().getId());
-    }
-
-    @Override
-    public void onError(Throwable throwable) {
-        Log.e("AmplifyGetStarted", throwable.toString());
-    }
-});
+    Amplify.API.mutate(task,
+    MutationType.CREATE,
+    taskCreationResponse -> Log.i("AmplifyGetStarted", "Added task with id: " + taskCreationResponse.getData().getId()),
+    apiFailure -> Log.e("AmplifyGetStarted", apiFailure.getMessage(), apiFailure)
+);
 ```
 
 c. Next query the results from your API:
 
 ```java
-Amplify.API.query(Task.class, new ResultListener<GraphQLResponse<Iterable<Task>>>() {
-    @Override
-    public void onResult(GraphQLResponse<Iterable<Task>> iterableGraphQLResponse) {
-        for(Task task : iterableGraphQLResponse.getData()) {
-            Log.i("AmplifyGetStarted", "Task : " + task.getTitle());
+Amplify.API.query(Task.class,
+    queryResults -> {
+        for(Task task : queryResults.getData()) {
+            Log.i("AmplifyGetStarted", task.getTitle());
         }
-    }
-
-    @Override
-    public void onError(Throwable throwable) {
-        Log.e("AmplifyGetStarted", throwable.toString());
-    }
-});
+    },
+    apiFailure -> Log.e("AmplifyGetStarted", apiFailure.getMessage(), apiFailure)
+);
 ```
 
 d. Finally, you can listen to the Subscription with a `StreamListener` using the `onNext` callback:
 
 ```java
-Amplify.API.subscribe(
-    Task.class,
+Amplify.API.subscribe(Task.class,
     SubscriptionType.ON_CREATE,
-    new StreamListener<GraphQLResponse<Task>>() {
-        @Override
-        public void onNext(GraphQLResponse<Task> taskGraphQLResponse) {
-            Log.i("AmplifyGetStarted", "Subscription detected a create: " +
-                    taskGraphQLResponse.getData().getTitle());
-        }
-
-        @Override
-        public void onComplete() {
-            // Whatever you want it to do on completing
-        }
-
-        @Override
-        public void onError(Throwable throwable) {
-            Log.e("AmplifyGetStarted", throwable.toString());
-        }
-    }
+    subscriptionId -> Log.i("AmplifyGetStarted", "Subscription established: "+subscriptionId),
+    taskCreated -> Log.i("AmplifyGetStarted", "Task created: "+taskCreated.getData().getTitle()),
```

Comment:
(Add one space on either side of the ` + `; `foo+bar` --> `foo + bar`

^^ Same for any uses